src/utils/DateUtils.ts

```diff
@@ -0,0 +1,9 @@
+const formatDate = (date: string) => {
+  if (typeof date !== 'undefined' && date) {
```

Comment:
Since you defined the date as `string` TypeScript will not allow undefined here. It's safe to remove. Also, would be great to have documentation on this function, and some unit testing as well :)